packages/lit-element/rollup.config.js

```diff
@@ -35,10 +36,5 @@ export default litProdConfig({
     {
       file: 'platform-support',
     },
-    {
-      file: 'lit-element',
-      output: 'lit.min',
-      name: 'Lit',
```

Comment:
You moved the `lit.min.js` bundle from `lit-element` to `lit`, but the bundle tests are still in `lit-element`, which is the cause of the CI failure AFAICT?

Reply:
fixed